### PR TITLE
Localize “Hello Minikube” into Italian

### PR DESCRIPTION
**Pull request by fabriziopandini**
Fixes: https://github.com/kubernetes/website/issues/20756

/assign @sftim 
/cc @Fale 
/cc @micheleberardi

### CODE REVIEW

**Comment by Fale** [comment]
Very nice work, @fabriziopandini, I added some comments on things that I think can improve the already very good translation :-)

File: content/it/docs/tutorials/_index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+* [Example: Deploying Cassandra with Stateful Sets](/docs/tutorials/stateful-application/cassandra/)
+
+* [Running ZooKeeper, A CP Distributed System](/docs/tutorials/stateful-application/zookeeper/)
+
+## CI/CD Pipelines
+
+* [Set Up a CI/CD Pipeline with Kubernetes Part 1: Overview](https://www.linux.com/blog/learn/chapter/Intro-to-Kubernetes/2017/5/set-cicd-pipeline-kubernetes-part-1-overview)
+
+* [Set Up a CI/CD Pipeline with a Jenkins Pod in Kubernetes (Part 2)](https://www.linux.com/blog/learn/chapter/Intro-to-Kubernetes/2017/6/set-cicd-pipeline-jenkins-pod-kubernetes-part-2)
```
[the remark this answers is not in the document]
Comment (sftim) [reply content/it/docs/tutorials/_index.md:53]: When linking to external content that will not be localized by the Kubernetes project, I think it is OK to leave the title untranslated (possibly: add a translation as well, maybe in brackets?)

File: content/it/docs/tutorials/_index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+* [Run and Scale a Distributed Crossword Puzzle App with CI/CD on Kubernetes (Part 3)](https://www.linux.com/blog/learn/chapter/intro-to-kubernetes/2017/6/run-and-scale-distributed-crossword-puzzle-app-cicd-kubernetes-part-3)
+
+* [Set Up CI/CD for a Distributed Crossword Puzzle App on Kubernetes (Part 4)](https://www.linux.com/blog/learn/chapter/intro-to-kubernetes/2017/6/set-cicd-distributed-crossword-puzzle-app-kubernetes-part-4)
```
[the remark this answers is not in the document]
Comment (sftim) [reply content/it/docs/tutorials/_index.md:57]: When linking to external content that will not be localized by the Kubernetes project, I think it is OK to leave the title untranslated (possibly: add a translation as well, maybe in brackets?)

File: content/it/docs/tutorials/_index.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+* [Running ZooKeeper, A CP Distributed System](/docs/tutorials/stateful-application/zookeeper/)
+
+## CI/CD Pipelines
+
+* [Set Up a CI/CD Pipeline with Kubernetes Part 1: Overview](https://www.linux.com/blog/learn/chapter/Intro-to-Kubernetes/2017/5/set-cicd-pipeline-kubernetes-part-1-overview)
+
+* [Set Up a CI/CD Pipeline with a Jenkins Pod in Kubernetes (Part 2)](https://www.linux.com/blog/learn/chapter/Intro-to-Kubernetes/2017/6/set-cicd-pipeline-jenkins-pod-kubernetes-part-2)
+
+* [Run and Scale a Distributed Crossword Puzzle App with CI/CD on Kubernetes (Part 3)](https://www.linux.com/blog/learn/chapter/intro-to-kubernetes/2017/6/run-and-scale-distributed-crossword-puzzle-app-cicd-kubernetes-part-3)
```
[the remark this answers is not in the document]
Comment (sftim) [reply content/it/docs/tutorials/_index.md:55]: When linking to external content that will not be localized by the Kubernetes project, I think it is OK to leave the title untranslated (possibly: add a translation as well, maybe in brackets?)

File: content/it/docs/tutorials/hello-minikube.md [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+
+{{% capture lessoncontent %}}
+
+## Crea un Minikube cluster
+
+1. Click **Launch Terminal**
+
+    {{< kat-button >}}
+
+    {{< note >}}Se hai installato minikube localmente, esegui `minikube start`.{{< /note >}}
```
[the remark this answers is not in the document]
Comment (sftim) [reply content/it/docs/tutorials/hello-minikube.md:50]: To be clear, this refers to the _first_ “minikube”.

File: content/it/docs/tutorials/_index.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+weight: 60
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+Questa sezione della documentazione di Kubernetes contiene i tutorials.
+Un tutorial mostra come raggiungere un obiettivo più complesso di un singolo
+[task](/docs/tasks/). Solitamente un tutorial ha diverse sezioni, ognuna delle quali
+consiste in una sequenza di step.
```
[the remark this answers is not in the document]
Comment (fabriziopandini) [reply content/it/docs/tutorials/_index.md:13]: usato task, che il termine utilizzato nella frase precedente

File: content/it/docs/tutorials/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fabriziopandini) [reply content/it/docs/tutorials/_index.md:53]: ok, this makes sense for external links

File: content/it/docs/tutorials/hello-minikube.md [hunk 6]
```diff
@@ -0,0 +1,279 @@
...
+---
+
+{{% capture overview %}}
+
+Questo tutorial mostra come eseguire una semplice applicazione in Kubernetes
+utilizzando [Minikube](/docs/setup/learning-environment/minikube) e Katacoda.
+Katacoda permette di operare su un'installazione di Kubernetes dal tuo browser.
+
+{{< note >}}
+Come alternativa, è possibile eseguire questo tutorial [installando minikube](/docs/tasks/tools/install-minikube/) localmente.
```
[the remark this answers is not in the document]
Comment (fabriziopandini) [reply content/it/docs/tutorials/hello-minikube.md:23]: fixed everywhere (except fo commands)

File: content/it/docs/tutorials/hello-minikube.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (fabriziopandini) [reply content/it/docs/tutorials/hello-minikube.md:50]: 👍

File: content/it/docs/tutorials/hello-minikube.md [hunk 7]
```diff
@@ -0,0 +1,279 @@
...
+
+```shell
+minikube delete
+```
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+* Approfondisci la tua conoscenza dei [Deployment objects](/docs/concepts/workloads/controllers/deployment/).
```
[the remark this answers is not in the document]
Comment (fabriziopandini) [reply content/it/docs/tutorials/hello-minikube.md:275]: Deployment in questo contesto è un nome API, non deve essere tradotto.
Ho rimosso objects

File: content/it/docs/tutorials/hello-minikube.md [hunk 8]
```diff
@@ -0,0 +1,279 @@
...
+* Approfondisci la tua conoscenza di [Deploying applications](/docs/tasks/run-application/run-stateless-application-deployment/).
+* Approfondisci la tua conoscenza dei [Service objects](/docs/concepts/services-networking/service/).
```
[the remark this answers is not in the document]
Comment (fabriziopandini) [reply content/it/docs/tutorials/hello-minikube.md:277]: Service in questo contesto è un nome API, non deve essere tradotto.
Ho rimosso objects

**Comment by fabriziopandini** [comment]
@Fale @sftim thanks for the feedback!
Comment addressed

**Comment by Fale** [comment]
Squash it, and it's a go for me!
I'll try to put
/lgtm
but I think the bot will complain about this

**Comment by fabriziopandini** [comment]
>  I think the bot will complain about this

As soon as you got your membership this should be fixed

**Comment by fabriziopandini** [comment]
@micheleberardi @sftim PTAL (unfortunately there are few reviewers/approvers as of today, and I can't review this by myself)

**Comment by Fale** [comment]
I can now leave the:
/lgtm 
:-)
Aside from this, maybe also @mattiaperi could have a look at this to be sure we can push it in soon :-)

File: content/it/docs/tutorials/_index.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+Questa sezione della documentazione di Kubernetes contiene i tutorials.
+Un tutorial mostra come raggiungere un obiettivo più complesso di un singolo
+[task](/docs/tasks/). Solitamente un tutorial ha diverse sezioni, ognuna delle quali
+consiste in una sequenza di più task.
+Prima di procedere con vari tutotial, raccomandiamo di aggiungere il
```
Comment (mattiaperi) [inline content/it/docs/tutorials/_index.md:14]: "tutotial" :)

File: content/it/docs/tutorials/hello-minikube.md [hunk 10]
```diff
@@ -0,0 +1,280 @@
...
+
+4. Katacoda environment only: Inserisci `30000`, a seguire click su **Display Port**.
+
+## Crea un Deployment
+
+Un Kubernetes [*Pod*](/docs/concepts/workloads/pods/pod/) è un gruppo di uno o più Containers,
+che sono uniti tra loro dal punto di vista amministrativo e che condividono lo stesso network.  
+Il Pod in questo tutorial ha un solo Container. Un Kubernetes
+[*Deployment*](/docs/concepts/workloads/controllers/deployment/) monitora lo stato del Pod ed
+eventualment provvedere a farlo ripartire nel caso questo termini. L'uso dei Deployments è la
```
Comment (mattiaperi) [inline content/it/docs/tutorials/hello-minikube.md:69]: eventualment*e*

**Comment by mattiaperi** [comment]
You are doing an awesome work! Apart for 2 small typos, I can't wait to see the pages published :)

**Comment by fabriziopandini** [comment]
@mattiaperi PTAL

**Comment by Fale** [comment]
/lgtm to me too
@mattiaperi can you please add `/approve`? ;-)

**Pull request merged**